tests/unit_tests/new/Collections/BranchesTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Collections;
+
+use Terminus\Models\Site;
+use Terminus\Collections\Branches;
+
+/**
+ * Testing class for Terminus\Collections\Branches
+ */
+class BranchesTest extends CollectionTestCase
+{
+    /**
+     * @inheritdoc
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        $this->site = $this->getMockBuilder(Site::class)
+          ->disableOriginalConstructor()
+          ->getMock();
+        $this->collection = new Branches(['site' => $this->site,]);
+        $this->collection->setRequest($this->request);
+    }
+
+    /**
+     * Tests Branches::fetch($options)
+     */
+    public function testFetch()
+    {
+        $data = ['name' => 'sha', 'name2' => 'sha2',];
+        $this->collection->fetch(compact('data'));
+    }
```

Comment:
It's not clear to me what this is testing. Does this throw an exception if fetch() does not receive the given data?